packages/keyring-api/src/eth/v2/eth-keyring-wrapper.ts

```diff
@@ -0,0 +1,213 @@
+import { TransactionFactory, type TypedTxData } from '@ethereumjs/tx';
+import type { MessageTypes, TypedMessage } from '@metamask/eth-sig-util';
+import { SignTypedDataVersion } from '@metamask/eth-sig-util';
+import type { EthKeyring } from '@metamask/keyring-utils';
+import { assert } from '@metamask/superstruct';
+import { add0x, type Hex, type Json } from '@metamask/utils';
+
+import type { KeyringAccount } from '../../api/account';
+import type { KeyringRequest } from '../../api/request';
+import {
+  KeyringWrapper,
+  type KeyringWrapperOptions,
+} from '../../api/v2/wrapper/keyring-wrapper';
+import {
+  EthDecryptParamsStruct,
+  EthGetAppKeyAddressParamsStruct,
+  EthPersonalSignParamsStruct,
+  EthSignEip7702AuthorizationParamsStruct,
+  EthSignParamsStruct,
+  EthSignTransactionParamsStruct,
+  EthSignTypedDataParamsStruct,
+  EthSignTypedDataV1ParamsStruct,
+} from '../rpc';
+import { EthMethod } from '../types';
+
+/**
+ * Additional Ethereum methods supported by Eth keyrings that are not in the standard EthMethod enum.
+ * These are primarily encryption and utility methods.
+ */
+export enum EthKeyringMethod {
+  Decrypt = 'eth_decrypt',
```

Comment:
Actually missed that one with the previous PR, but this one should `eth_decryptMessage` and `DecryptMessage` instead no?

Reply:
Yup, I got confused by the actual method name 🙈 the `KeyringController` also uses `decryptMessage`, but let's keep `eth_decrypt` 👍

I wonder if it would be worth to add jsdoc + `@deprecated` for the enum member? 🤔